test/server/routes/admin.test.js

```diff
@@ -1,7 +1,9 @@
 import chaiHttp from 'chai-http';
```

Comment:
Did you write a test for a user that is not an admin trying to access this route?

Reply:
No, I didn't. I only wrote a test to target the enforceVerification function that I worked on. I thought the person that wrote the authorizeAdmin middleware was supposed to write a test for it. I will do that now though since you asked.